pages/settings/devices.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <v-row class="mx-0 justify-space-between">
+          <h2 class="text-h6 mb-2">
+            {{ $t('settings.devices.devices') }}
+          </h2>
+          <div>
+            <v-btn
+              v-if="devices"
+              color="error"
+              class="ml-a"
+              @click="deleteAllDevices"
+            >
+              {{ $t('settings.devices.deleteAll') }}
+            </v-btn>
+            <v-btn
+              ref="noreferrer noopener"
+              href="https://jellyfin.org/docs/general/server/devices.html"
+              target="_blank"
+            >
+              {{ $t('settings.help') }}
+            </v-btn>
+          </div>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <v-card>
+          <v-list v-if="devices">
+            <v-list-item
+              v-for="device in devices"
+              :key="device.Id"
+              @click="setSelectedDevice(device)"
+            >
+              <v-avatar></v-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="device.Name" />
+                <v-list-item-title
+                  v-text="`${device.AppName} - ${device.AppVersion}`"
+                />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-menu>
+                  <template #activator="{ on, attrs }">
+                    <v-btn icon v-bind="attrs" v-on="on" @click.stop.prevent>
+                      <v-icon>mdi-dots-horizontal</v-icon>
+                    </v-btn>
+                  </template>
+                  <v-list>
+                    <v-list-item @click="deleteDevice(device)">
+                      <v-list-item-avatar>
+                        <v-icon>mdi-delete</v-icon>
+                      </v-list-item-avatar>
+                      <v-list-item-title>Delete</v-list-item-title>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </v-list-item-action>
+            </v-list-item>
+          </v-list>
+          <v-card v-else>
+            <v-card-title>
+              {{ $t('settings.devices.noDevicesFound') }}
+            </v-card-title>
+          </v-card>
+        </v-card>
+      </v-col>
+      <!-- Item Info (Large Screens) -->
+      <v-col v-if="$vuetify.breakpoint.lgAndUp">
```

Comment:
```suggestion
      <!-- Item Info (Tablet/Laptop Screens) -->
      <v-col v-if="$vuetify.breakpoint.mdAndUp">
```
I've reduced the breakpoint so the dialog doesn't appear on largish screens, it looks like this at the breakpoint with this suggestion: 
![Screenshot_2021-01-12 Jellyfin](https://user-images.githubusercontent.com/1619359/104374195-ce65ec80-5521-11eb-9d34-9d428802fa5f.png)

Reply:
Sounds good, I'll update it.